tests/misc/CachingTest.py

```diff
@@ -146,13 +146,11 @@ def test_caching_multi_results(self):
                 retval, output = execute_coala(
                    coala.main,
                    'coala',
-                   '-c', filename + '.coafile',
-                   '-f', filename + 'test.py')
+                   '-c', filename + '.coafile')
```

Comment:
As discussed, this is a regression.
previously `coala -f abc.py -c .coafile` would check `abc.py` using bears in section `LineCounting `, but after this PR it would not.
This presumably only occurred if `files` was empty in `LineCounting` and `default`.  Still need to verify that.
This is possible a small regression and undesirable to support, even for the short term.
More analysis needed, to be very confident what exactly is being de-supported (It should be relnoted), and that it is unlikely anyone needed this.